src/main/java/com/arpnetworking/tsdcore/sinks/KairosDbSink.java

```diff
@@ -91,6 +120,15 @@ public Object toLogValue() {
             serializer.serializeDatum(completeChunks, currentChunk, chunkStream, datum);
         }
 
+        if (_publishHistograms && histogram != null) {
```

Comment:
So just to be clear that publishing histograms publishes both the histogram and the normal datum (see line 120). Can you add a flag to control normal publication as well so we can effect a full transition from normal to both to histogram.